src/app/pages/CChainPages/CChainPage.tsx

```diff
@@ -13,6 +13,14 @@ import { Status } from 'types'
 import LoadingWrapper from 'app/components/LoadingWrapper'
 
 export default function CChainPage() {
+    const [componentLoaded, setComponentLoaded] = React.useState(false)
```

Comment:
maybe setTimout needs to be refactored or search for another solution
* metaData is inside the state
* loading c-chain details is done after the chains is loaded
* the components on this page have their own loading
* setTimout can be a problem if the connexion of the user is bad (400ms is not enough)